obspy/io/sac/sactrace.py

```diff
@@ -1231,7 +1248,7 @@ def _from_arrays(cls, hf=None, hi=None, hs=None, data=None):
         .. rubric:: Example
 
         >>> sac = SACTrace._from_arrays()
-        >>> print(sac)  # doctest: +NORMALIZE_WHITESPACE +SKIP
+        >>> print(sac)  # doctest: +NORMALIZE_WHITESPACE
```

Comment:
If this test is giving the PR problems, and it was previously skipped, maybe just put the `+SKIP` back in and I'll make a new PR for it.